galata/test/jupyterlab/notebook-scroll.test.ts

```diff
@@ -131,15 +124,53 @@ test.describe('Notebook scroll on execution', () => {
     // The third cell should be positioned at the bottom, revealing between 10 to 20% of its content.
     await expect(thirdCellLocator).toBeInViewport({ ratio: 0.1 });
     await expect(thirdCellLocator).not.toBeInViewport({ ratio: 0.2 });
+    await expect(thirdCellLocator).not.toBeInViewport({ ratio: 0.2 });
```

Comment:
This line is duplicated

Reply:
Great catch, thanks! It now looks good for me:

```diff
20c20
<       windowingMode: 'full'
---
>       windowingMode: 'none'
25c25
< test.describe('Notebook scroll on navigation (with windowing)', () => {
---
> test.describe('Notebook scroll on navigation (no windowing)', () => {
65c65
< test.describe('Notebook scroll on execution (with windowing)', () => {
---
> test.describe('Notebook scroll on execution (no windowing)', () => {
172c172
< test.describe('Notebook scroll over long outputs (with windowing)', () => {
---
> test.describe('Notebook scroll over long outputs (no windowing)', () => {
```